### PR TITLE
add neo-hpstr blog theme

### PR DESCRIPTION
**Pull request by aron-bordin**

### CODE REVIEW

File: _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Neo-HPSTR Jekyll Blog template
+date: 2015-09-05 12:00:00
+homepage: http://github.com/aron-bordin/neo-hpstr-jekyll-theme
+download: http://github.com/aron-bordin/neo-hpstr-jekyll-theme/archive/master.zip
+demo: http://aron-bordin.github.io/neo-hpstr-jekyll-theme/
+author: Aron Bordin
+thumbnail: neo-hpstr.png
+license: MIT
```
Comment (gynter) [inline _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown]: Hello! Please change the license string from `MIT` to `MIT License` so the formatting will be similar to all other themes.

File: _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+license_link: https://raw.githubusercontent.com/aron-bordin/neo-hpstr-jekyll-theme/master/LICENSE
+---
+### Neo-HPSTR Jekyll Template
```
Comment (gynter) [inline _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown]: Please add an empty line after front mater declaration and add empty lines after each heading.

File: _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+<iframe src="//ghbtns.com/github-btn.html?user=aron-bordin&repo=neo-hpstr-jekyll-theme&type=watch&count=true&size=large" allowtransparency="true" frameborder="0" scrolling="0" width="160px" height="30px"></iframe>
+
+
+
+
+
```
Comment (gynter) [inline _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown]: Please remove excessive newlines.

**Comment by aron-bordin** [comment]
Hi @gynter , PR updated

File: _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Neo-HPSTR Jekyll Blog template
```
Comment (gynter) [inline _posts/2015-09-05-neo-hpsrt-jekyll-template.markdown]: One more thing, is the theme name "Neo-HPSTR Jekyll Blog template" or "Neo-HPSTR Jekyll template"? If the latter, then change the title please and rename thumbnail image to neo-hpsrt-jekyll-template.png, i.e keep the thumbnail filename same as the template post, but without the date prefix.

**Comment by aron-bordin** [comment]
@gynter updated :)

**Comment by gynter** [comment]
Nice thanks!

@mattvh :shipit:

**Comment by geraldb** [comment]
@aron-bordin  While you're waiting to get accepted on jekyllthemes.org - I invite you to send a pull request to the alternative [Dr. Jekyll's Themes](http://drjekyllthemes.github.io) directory. Same system. Just a little simpler, easier. Your Neo-HPSTR theme is welcome. Cheers.

**Pull request merged**